src/main/java/org/cryptomator/ui/addvaultwizard/CreateNewVaultLocationController.java

```diff
@@ -196,6 +214,10 @@ public void back() {
 	@FXML
 	public void next() {
 		if (validVaultPath.getValue()) {
+			Path parentPath;
+			if (this.getVaultPath() != null && (parentPath = this.getVaultPath().getParent()) != null) {
```

Comment:
don't mix conditions, expressions and assignments in the same line. First make the assignment, then check it with the condition.